main.go

```diff
@@ -113,17 +122,19 @@ func (c *transipDNSProviderSolver) Present(ch *v1alpha1.ChallengeRequest) error
 	// This method should tolerate being called multiple times
 	// with the same value. If a TXT record for this request
 	// already exists, we'll simply exit.
-	for _, s := range domainInfo.DNSEntries {
+	for _, s := range dnsEntries {
 		if s == acmeDnsEntry {
+			fmt.Printf("ACME DNS entry already exists, skip\n")
 			return nil
 		}
 	}
 
 	// Record does not exist - append it to the existing DNS entries
-	domainInfo.DNSEntries = append(domainInfo.DNSEntries, acmeDnsEntry)
+	dnsEntries = append(dnsEntries, acmeDnsEntry)
```

Comment:
This line has become redundant now that `gotransip` has a `AddDNSEntry` function. Its only use was to compose the complete set of DNS records, as the complete set had to be passed on to TransIP.

Could you remove this line?

Reply:
You're right. Removed.